service/controller/chart/resource/release/update.go

```diff
@@ -243,10 +244,16 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 	}
 
 	if isReleaseModified(currentReleaseState, desiredReleaseState) {
-		r.logger.Debugf(ctx, "the %#q release has to be updated", desiredReleaseState.Name)
+		// Ignoring `Values` among diff since it could contain secret data.
```

Comment:
```suggestion
		// Ignoring `Values` in diff since it could contain secret data and we use MD5 hash for comparison.
```

Could we go with this?

Reply:
Nice :)